docs/netdata-agent/versions-and-platforms.md

```diff
@@ -1,116 +1,235 @@
-| Redhat Enterprise Linux |             8.x, 9.x             |               `x86_64`, `AArch64`                |     RPM      |
-|         Ubuntu          |       20.04, 22.04, 23.10        |       `x86_64`, `i386`, `ARMv7`, `AArch64`       |     DEB      |
-
-> IMPORTANT: Linux distributions frequently provide binary packages of Netdata. However, the packages you will find in the distributions' repositories may be outdated, incomplete, missing significant features or completely broken. We recommend using the packages we provide.
+| **Docker under Linux** |         19.03 and later          | `x86_64`, `i386`, `ARMv7`, `AArch64`, `POWER8+`  | docker image |
+|   **Static Builds**    |                -                 | `x86_64`, `ARMv6`, `ARMv7`, `AArch64`, `POWER8+` |   .gz.run    |
+|    **Alma Linux**      |             8.x, 9.x             |               `x86_64`, `AArch64`                |     RPM      |
+|   **Amazon Linux**     |             2, 2023              |               `x86_64`, `AArch64`                |     RPM      |
+|      **Centos**        |               7.x                |                     `x86_64`                     |     RPM      |
+|       **Debian**       |         10.x, 11.x, 12.x         |       `x86_64`, `i386`, `ARMv7`, `AArch64`       |     DEB      |
+|       **Fedora**       |            37, 38, 39            |               `x86_64`, `AArch64`                |     RPM      |
+|      **OpenSUSE**      | Leap 15.4, Leap 15.5, Tumbleweed |               `x86_64`, `AArch64`                |     RPM      |
+|    **Oracle Linux**    |             8.x, 9.x             |               `x86_64`, `AArch64`                |     RPM      |
+| **Redhat Enterprise Linux** |               7.x                |                     `x86_64`                     |     RPM      |
+| **Redhat Enterprise Linux** |             8.x, 9.x             |               `x86_64`, `AArch64`                |     RPM      |
+|       **Ubuntu**       |       20.04, 22.04, 23.10        |       `x86_64`, `i386`, `ARMv7`, `AArch64`       |     DEB      |
+
+:::important  
+
+Linux distributions frequently provide binary packages of Netdata. However, the packages you will find in the distributions' repositories **may be outdated, incomplete, missing significant features or completely broken**. We recommend using the packages we provide.
+
+:::
 
 ## Third-party Supported Binary Packages
 
 The following distributions always provide the latest stable version of Netdata:
 
 |  Platform  | Platform Versions |    Released Packages Architecture    |
 |:----------:|:-----------------:|:------------------------------------:|
-| Arch Linux |      Latest       | All the Arch supported architectures |
-| MacOS Brew |      Latest       | All the Brew supported architectures |
+| **Arch Linux** |      Latest       | All the Arch supported architectures |
+| **MacOS Brew** |      Latest       | All the Brew supported architectures |
```

Comment:
We should remove Homebrew here, they stopped publishing updated Netdata packages a while back.